RHEL5 still does not use bg_itable_unused_lo/hi. This patch
can be removed from the series once ext4_itable_unused_count()
is introduced in the RHEL5 mainline.

Index: linux-2.6.18-128.1.6/fs/ext4/ext4.h
===================================================================
--- linux-2.6.18-128.1.6.orig/fs/ext4/ext4.h
+++ linux-2.6.18-128.1.6/fs/ext4/ext4.h
@@ -1275,6 +1275,11 @@ static inline unsigned int ext4_flex_bg_
 	return 1 << sbi->s_log_groups_per_flex;
 }
 
+static inline __u32 ext4_itable_unused_count(struct super_block *sb,
+                                             struct ext4_group_desc *bg) {
+	       return le16_to_cpu(bg->bg_itable_unused);
+}
+
 #define ext4_std_error(sb, errno)				\
 do {								\
 	if ((errno))						\
